static/index.html

```diff
@@ -49,31 +49,42 @@
       </div>
       <div class="container">
         <div class="row">
-          <div class="col-12 col-sm-12 col-md-12 col-lg-6">
+          <div class="col-12 col-md-12 col-lg-6">
             <div class="logo-wrapper">
               <div class="logo"></div>
             </div>
           </div>
-        <div class="description  col-lg-6">
-          <div class="col-12 col-sm-12 col-md-12">
-            <h1>Conference about Open Source</h1>
-              <p>The 5th edition of the Drupalcamp Cebu is in the making. We will plan exciting sessions and tasks about the Drupal, PHP, DevOPS, UX and more. To recieve the news about the event before everyone else, subscribe to our news letter below.</p>
-            <form>
-              <div class="form-wrapper col-12">
-                <input type="text" placeholder="Email address" class="input-text col-7">&emsp;
-                <input type="submit" value="Submit" class="submit-btn col-4">
-              </div>
-            </form>
+          <div class="description col-md-12 col-lg-6">
+            <div class="">
```

Comment:
<div class="">
Why empty class? Maybe this can be removed?

Reply:
Done